README.md

```diff
@@ -12,26 +12,29 @@ This project provides a Python API for working with [C2PA](https://c2pa.org/) (C
 ## Project Structure
 
 ```bash
-python_api/ 
-    ├── examples/        # Example scripts demonstrating usage
-    ├── src/            # Source code for the C2PA Python API
-    │   └── c2pa/      # Main package directory
-    │       └── libs/  # Platform-specific libraries
-    ├── tests/          # Unit tests and benchmarks
-    ├── artifacts/      # Platform-specific libraries for building
-    │   ├── win_amd64/     # Windows x64 libraries
-    │   ├── win_arm64/     # Windows ARM64 libraries
-    │   ├── macosx_x86_64/ # macOS x64 libraries
-    │   ├── macosx_arm64/  # macOS ARM64 libraries
-    │   ├── linux_x86_64/  # Linux x64 libraries
-    │   └── linux_aarch64/ # Linux ARM64 libraries
-    ├── requirements.txt # Python dependencies
-    └── README.md       # Project documentation
+.
+├── .github/           # GitHub configuration files
+├── artifacts/         # Platform-specific libraries for building (per subfolder)
+│   └── linux_x86_64/ # Linux ARM64 libraries
```

Comment:
I think this could be any of the platforms, but this should give the general idea

Reply:
Fix incoming in next push! (+further docs improvements)